src/components/features/cart/cartList/CartList.tsx

```diff
@@ -0,0 +1,37 @@
+import SelectBox from '../../../common/selectBox/SelectBox';
+import CartItem from '../cartItem/CartItem';
+import * as S from './CartList.styles';
+import { CartItemType } from '../types';
+import { useCartSelectionContext } from '../contexts/CartSelectionContext';
+
+interface CartListProps {
+  cartItems: CartItemType[];
+  refetch: () => Promise<void>;
+}
+
+function CartList({ cartItems, refetch }: CartListProps) {
+  const selection = useCartSelectionContext();
+  return (
+    <S.Container>
+      <S.AllSelectBox>
+        <SelectBox
+          id="allSelect"
+          selected={selection.allSelected}
+          onClick={selection.toggleAll}
+        />
+        <S.AllSelectText htmlFor="allSelect">전체선택</S.AllSelectText>
+      </S.AllSelectBox>
+      {cartItems.map((cartItem, index) => (
+        <CartItem
+          key={cartItem.id}
+          cartItem={cartItem}
```

Comment:
```suggestion
          {...cartItem}
```
이렇게 전개해서 넘겨주면 어떨까요?
이러면 props를 객체가 아니라 값으로 넘겨주게 되는데,
이게 나중에 최적화할 때 큰 도움이 된답니다 ㅎㅎ (어떤 도움이 되는지 알아보시면 좋답니다!)
무엇보다 컴포넌트 입장에서도 편해요.